### DISCUSSION TITLE
Is it possible to create lexer for _hyperscript?

### DISCUSSION THREAD

**Discussion by ghost** [orbitalquark/textadept#352] [Q&A]
https://hyperscript.org/docs/

https://github.com/gnat/hyperscript-sublime/

https://github.com/dz4k/vscode-hyperscript

**Comment by orbitalquark** [comment]
That looks doable using embedded lexers. The child would be the 
hyperscript within quotes, the start rule would probably be =" and the 
end rule would probably be "
